src/main/menu_app/src/components/Navbar.js

```diff
@@ -1,25 +1,24 @@
 import React from "react";
-import { NavLink } from "react-router-dom";
+import { Link } from "react-router-dom";
 
 import './../App.css';
 
 function Navbar() {
   return (
-  <div className='container'>
-    <nav className='navbar'>
-      <ul className='navList'>
-        <li>
-          <NavLink href="/" className='navItem'>Home</NavLink>
+    <nav className='navbar bg-body-tertiary px-3 mb-3'>
+      <h1 className="navbar-brand">Menu Generator</h1>
+      <ul className='nav justify-content-center'>
+        <li className="nav-item">
+          <Link to="/" className='nav-link'>Home</Link>
         </li>
-        <li>
-          <NavLink href="/categories" className='navItem'>Categories</NavLink>
+        <li className="nav-item">
+          <Link to="/categories" className='nav-link'>Categories</Link>
```

Comment:
What is the difference between `Link` and `NavLink`? You sure you needed to replace it?

Reply:
It depends. On this stage I do not use special features of NavLink tag. I'm thinking about coloring the active tab's name, and then I may need it back...

Reply:
In that case, I'll leave the decision up to you. You either should:
- change the commit message to address this swap as not significant
- revert `Link` to `NavLink`
- move change from `Link` to `NavLink` to another commit